tests/basic/bug68986.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Bug #68986 (pointer returned by php_stream_fopen_temporary_file not validated in memory.c)
+--INI--
+default_charset=UTF-8
+--FILE--
+<?php
+if (substr(PHP_OS, 0, 3) == 'WIN') {
+	die('skip.. only for unix');
+}
+
+$tempDir = getenv("TMPDIR");
+mkdir($tempDir . '/php68986');
+system("chmod 444 " . $tempDir . '/php68986');
+putenv("TMPDIR=" . $tempDir . '/php68986');
```

Comment:
you must missed the NEWS update commit https://github.com/php/php-src/compare/a79f332333...ae90010891 ,  as as I explained in the other thread, this is a simple null pointer dereference, and this test script will leave a un-rm-able directory in testing box, it's pretty  okey to not include one test script for one line change.